### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by Xcelled**

### CODE REVIEW

File: uxexperience/settings.py [hunk 1]
```diff
@@ -106,8 +105,11 @@
 ]
 
 # Heroku DB configuration
-db_from_env = dj_database_url.config(conn_max_age=500)
-DATABASES['default'].update(db_from_env)
+try: import dj_database_url
```
Comment (kalebdavis) [inline uxexperience/settings.py]: What's this about?

Comment (Xcelled) [inline uxexperience/settings.py]: it's so I can run it locally without heroku. It just skips importing heroku stuff

Comment (kalebdavis) [inline uxexperience/settings.py]: okay then looks good

Comment (kalebdavis) [inline uxexperience/settings.py]: but if you run pip install -r requirements.txt in your virtualenv you should install those packages so it shouldn't be an issue.

**Pull request merged**